src/FNA3D_Driver_Vulkan.c

```diff
@@ -2060,6 +2069,20 @@ static uint8_t VULKAN_INTERNAL_QuerySwapChainSupport(
 	SwapChainSupportDetails *outputDetails
 ) {
 	VkResult result;
+	VkBool32 supportsPresent;
+
+	renderer->vkGetPhysicalDeviceSurfaceSupportKHR(
```

Comment:
I think we check this somewhere else... does one make the other redundant by chance?

Reply:
This gets checked when we do the dummy surface in PrepareWindowAttributes, but we have to check it for every surface we create or Vulkan validation yells at us.